README.md

```diff
@@ -1,17 +1,63 @@
 ﻿# Slipstream
 
-**⚠ This project is under heavily development and should be considered alpha
+**⚠ This project is under early and experimental development and should be considered alpha
 software ⚠**
 
-## What's Slipstream?
-
-Slipstream is an event-based application, that was built to connect or act on
-events from [IRacnig](http://www.iracing.com).
-
-A number of Slipstream plugins generates event or acts on events. As user of
-Slipstream you can provide Lua scripts that interacts with these events as
-well. This allows you to write Lua scripts that records laptimes to a file,
-output messages to twitch, play samples or use text-to-speech to notify you.
+## What is Slipstream?
+
+Slipstream is an application with an event-based architecture that allows 
+users to programatically handle events using Lua scripts. The primary
+objective for the application is to provide a dynamic and flexible solution
+to act on [iRacing](http://www.iracing.com) telemetry events.
+
+The extensibility of the core application can easily accomodate other types
+of plugins to create a comprehensive solution for iRacers and streamers. 
+
+![Slipstream Screenshot](docs/img/screen_shot.PNG)
+
+Other plugin examples are: 
+- Twitch chat
+- User interaction (button click)
+- File monitor
+- Lua scripts
+
+## How it works
+
+Events raised by the plugins are sent to Lua scripts where the events can be translated
+to real world actions like:
+- Interacting with IoT devices
+- Playing audio samples
+- Text to speech
+
+Example script
+```Lua
+local whitelist = {}
+
+whitelist["sassy_mcsassypants"] = true
+whitelist["larryrabbets"] = true
```

Comment:
Instead of setting these to true, you could use the name you want to use and then you can remove `say_name`

Reply:
yup, was just a quick hack while Sassy was testing, we probably need better examples in the readme and then examples in the scripts folder